aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -190,6 +216,48 @@ module aptos_framework::account {
         account_resource.authentication_key = new_auth_key;
     }
 
+    public entry fun rotate_authentication_key_ed25519(account: &signer, new_public_key: vector<u8>, signature: vector<u8>, scheme: vector<u8>) acquires Account, OriginatingAddress {
```

Comment:
swap the api: signature and then public key
the motivation being that other verify apis do the same.

Reply:
also eliminate the scheme as that's already defined by the ed25519 on the function name